chef_master/source/auth.rst

```diff
@@ -81,16 +81,15 @@ Each workstation stores its private key in the chef-repo. This private key is ge
 
 .. tag chef_repo_description
 
-The chef-repo is a directory on your workstation that stores:
+The chef-repo is a directory on your workstation that stores everything you need to define your infrastructure with Chef Infra:
 
 * Cookbooks (including recipes, attributes, custom resources, libraries, and templates)
-* Roles
 * Data bags
-* Environments
+* Policyfiles
 
 The chef-repo directory should be synchronized with a version control system, such as git. All of the data in the chef-repo should be treated like source code.
 
-knife is used to upload data to the Chef Infra Server from the chef-repo directory. Once uploaded, that data is used by Chef Infra Client to manage all of the nodes that are registered with the Chef Infra Server and to ensure that the correct cookbooks, environments, roles, and other settings are applied to nodes correctly.
+The chef and knife commands are used to upload data to the Chef Infra Server from the chef-repo directory. Once uploaded, that data is used by Chef Infra Client to manage all of the nodes that are registered with the Chef Infra Server and to ensure that the correct cookbooks, policyfiles, and other settings are applied to nodes correctly.
```

Comment:
```suggestion
You'll use the ``chef`` and ``knife ``commands to upload data to the Chef Infra Server from the chef-repo directory. Once uploaded, Chef Infra Client uses that data to manage the nodes registered with the Chef Infra Server and to ensure that it applies the right cookbooks, policyfiles, and settings to the right nodes in the right order.
```

Does the repetition of "right" work for you? Is it clarifying or distracting? (I'm on the fence)

Reply:
double right seems kinda odd

Reply:
I was thinking about doing something in the right place, at the right time, in the right order. Rhetorically sound, but perhaps the wrong audience.